front/src/components/Applications/ApplicationList.tsx

```diff
@@ -142,22 +162,58 @@ const ApplicationList = () => {
         updatedData
       );
 
+      setUniqueAlert("");
+
       if (response) {
-        setRows(response.data);
+        if (response.statusCode == 409) {
+          setUniqueAlert(response.message);
+        } else if (response.statusCode == 200) {
+          handleEditModalClose();
+          setRows(response.data);
+          setAlertShow(response.message);
+          setTimeout(() => {
+            setAlertShow("");
+          }, 3000);
+        }
       }
     } catch (error: any) {
+      var response = error.response.data;
+      if (response.statusCode == 422 && response.message.application) {
+        setUniqueAlert(response.message.application);
+      }
+
       console.log(error);
     }
   };
+
+  const handleDeleteConfirm = async (selectedRow: any) => {
+    if (selectedRow !== null) {
+      try {
+        const response = await ApplicationApi.deleteApplication(selectedRow.id);
+
+        if (response && response.data) {
+          setRows(response.data);
+          setAlertShow(response.message);
+          setTimeout(() => {
```

Comment:
Avoid using setTimeout in the application.

Reply:
setTimeout has removed